orchestrator/packages/api/src/controllers/grading-queue-controller.ts

```diff
@@ -120,6 +122,7 @@ export const createOrUpdateImmediateJob = async (
   try {
     const status = await putJobInQueue(req.body, true);
     logger.info(`New job from ${new URL(req.body.response_url).host} sent to ${status.location} with database id ${status.id}.`);
+    await touchGraderImageFile(req.body);
```

Comment:
risky -- I don't think a failure in `touch`ing the grader file should prevent the 200 status response!

Reply:
Fair enough -- will swap for a `.catch` and an inner warning log statement